pkgs/tools/networking/ppp/default.nix

```diff
@@ -34,7 +34,9 @@ stdenv.mkDerivation rec {
   '';
 
   postFixup = ''
-    substituteInPlace $out/bin/{pon,poff,plog} --replace "/usr/sbin" "$out/bin"
+    for tgt in pon poff plog; do
+      substituteInPlace "$out/bin/$tgt" --replace "/usr/sbin" "$out/bin"
```

Comment:
Do you know what the underlying error is here (I'll find it if you don't off-hand)? I didn't intend to change the interface of substituteInPlace so I should fix that rather than this package need be changed.